src/quo2/components/notifications/notification_dot.cljs

```diff
@@ -4,12 +4,18 @@
 
 (def ^:const size 8)
 
+(defn dot-background-color
+  [customization-color]
+  (if customization-color
+    (colors/custom-color-by-theme customization-color 50 60)
+    (colors/theme-colors colors/primary-50 colors/primary-60)))
```

Comment:
Should the notification dot be themeable and should we pass the current `theme` to `theme-colors` instead of relying on the user's theme?

Reply:
something to note, this function `colors/custom-color-by-theme` is not setup to work with new theming mechanism, I think for now it's better to use
```
(colors/theme-colors
 (colors/custom-color customization-color 50)
  (colors/custom-color customization-color 60)
  theme)
  ```
  
  and theme value should be gotten from a prop or probably easier again to pass using `with-theme` hoc similiar to other quo2 components